lib/util.js

```diff
@@ -100,7 +100,7 @@ function makeRequest(options, cb) {
         });
       }
     }
-    if (response.body != null && response.body != null) return cb(new KinveyError(response.body.code, response.body.description));
+    if (response.body != null && response.body != null) return cb(new KinveyError(response.body.code, response.body.description, response.body.debug));
```

Comment:
I know it's not your change but do you think you can remove one of the `response.body` evaluations? :satisfied: